public/usage-examples/geometry/cicle_intersects_triangle-1-example-opp.cs

```diff
@@ -0,0 +1,46 @@
+using SplashKitSDK;
+
```

Comment:
Program can't run due to missing namespace. Insert:


```suggestion
namespace CircleTriangleIntersectExample
{
```